tests/release_tests/test_launch/launch-config.yaml

```diff
@@ -0,0 +1,13 @@
+base_url: https://api.wandb.ai
+max_jobs: -1
+entity: launch-release-testing
+queues:
+- kubernetes-queue
+builder:
+  type: noop
+registry:
+  type: ecr
+  repository: release-testing
+environment:
+  type: aws
+  region: us-east-2
```

Comment:
nit: line at end of file. Also is there a reason this is a `yaml` file vs. `yml` ?

Reply:
Thanks, this file was leftover from a previous version.